docker/env/hydra.sh

```diff
@@ -160,9 +161,9 @@ fi
 
 if [[ ${USER} == "jenkins" || -z "`$tool images ${DOCKER_REPO}:${VERSION} -q`" ]]; then
     echo "Pull version $VERSION from Docker Hub..."
-    $tool pull ${DOCKER_REPO}:${VERSION}
+    $tool pull ${DOCKER_REGISTRY}/${DOCKER_REPO}:${VERSION}
```

Comment:
This is not enough

Later there are more usages of the REPO, in the docker/podman run command run

Reply:
Why is it not enough? I tried it with podman, and it worked! Can you please try?
It seems that after you pulled the image, it's in the local cache and can be found without the registry.

Reply:
seems that you are correct, couldn't understand why podman couldn't be aligned across all it's commands.